frontend/src/metabase/entities/containers/EntityListLoader.jsx

```diff
@@ -192,11 +234,16 @@ export default class EntityListLoader extends React.Component {
   };
 }
 
-export const entityListLoader = (ellProps: Props) =>
+EntityListLoader.propTypes = propTypes;
+EntityListLoader.defaultProps = defaultProps;
+
+export default EntityListLoader;
+
+export const entityListLoader = ellProps =>
   // eslint-disable-line react/display-name
-  (ComposedComponent: any) =>
+  ComposedComponent =>
     // eslint-disable-next-line react/display-name
```

Comment:
Maybe there is a quick way to remove the eslint disabler here.

Reply:
I'll take a look 👌